com.coala.core/plugin.xml

```diff
@@ -57,7 +57,7 @@
       <menuContribution
             locationURI="popup:org.eclipse.ui.popup.any?after=additions">
          <menu
-               icon="icons/coala-analyzer.gif"
+               icon="icons/coala.gif"
```

Comment:
In icons folder the file is named to be "coala-analyzer.gif" rename it to "coala.gif" might possibly fix the issue as well as ci failure!!